index.js

```diff
@@ -1,26 +1,19 @@
-/* eslint-disable max-classes-per-file */
-/* eslint-disable no-use-before-define */
-
-// eslint-disable-next-line no-unused-vars
-
-// Hidde and display sections
 import {
   addBooks,
   contact,
   addBoo,
   contactHandler,
 } from './modules/hideDisplay.js';
 
-// class declaration
-
 import { BookCollection } from './modules/bookCollection.js';
 
-import { DateTime } from './luxon/build/es6/luxon.js';
+import { DateTime } from './modules/luxon/build/es6/luxon.js';
```

Comment:
- [ ] Good job with the changes but it appears that luxon is not being used as an `npm package` but was added as a module. Kindly stick to the suggestion of the previous reviewer by installing `luxon` as a package and import it from the `node_modules` instead of going through the stress of dealing with the entire file. This what npm packages allow uu to save valuable time from writing from scratch :wink: